src/shared/tests.h

```diff
@@ -200,3 +200,43 @@ static inline int run_test_table(void) {
         DEFINE_TEST_MAIN_FULL(log_level, intro, NULL)
 #define DEFINE_TEST_MAIN(log_level)                     \
         DEFINE_TEST_MAIN_FULL(log_level, NULL, NULL)
+
+#define ASSERT_OK(expr)                                                                                 \
+        ({                                                                                              \
+                int result = (expr);                                                                    \
+                if (result < 0) {                                                                       \
+                        log_error_errno("Assertion failed: %s (result: %d, error: %m)", #expr, result); \
+                        abort();                                                                        \
+                }                                                                                       \
+        })
+
```

Comment:
extra newline

Reply:
thank you very much @bluca I made the changes.